clients/go-ethereum:local/geth.sh

```diff
@@ -64,6 +64,9 @@ fi
 echo "Initializing database with genesis state..."
 /geth $FLAGS init /genesis.json
 
+# Don't immediately abort, some imports are meant to fail
+set +e
+
```

Comment:
Could you detail where we have such scenarios. Notably, why would we add a block as a pre-state if we know it will fail. Shouldn't that block just be omitted since it's a noop anyway?